### PR TITLE
Google Cloud Compute - Service JSON to Credentials - DO NOT MERGE TO MASTER!!!

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25432

## Description
Change the service account Json parameter to a Credentials pamaeter

### CODE REVIEW

File: Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 """ GLOBALS/PARAMS """
 
 # Params for assembling object of the Service Account Credentials File Contents
-SERVICE_ACCOUNT_FILE = demisto.params().get('service')
+SERVICE_ACCOUNT_FILE = demisto.params().get('credentials').get('password')
```
Comment (yaakovi) [inline Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py:19]:
```suggestion
SERVICE_ACCOUNT_FILE = demisto.params().get('credentials', {}).get('password')
```

File: Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.yml [hunk 2]
```diff
@@ -3,10 +3,10 @@ commonfields:
   id: Google Cloud Compute
   version: -1
 configuration:
-- display: Service Account Private Key file contents (JSON)
-  name: service
-  required: true
-  type: 4
+- display: add Service Account Private Key file contents (JSON) to "Password"
+  name: credentials
+  required: false
```
Comment (yaakovi) [inline Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.yml:8]: why not `required`?

Comment (orlichter1) [reply Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.yml:8]: by accident. I meant for it to be required. I though I did it :)

**Comment by yaakovi** [comment]
closed this PR, reopen if needed

**Pull request closed without merging**